lms/djangoapps/instructor/views/api.py

```diff
@@ -2846,7 +2846,13 @@ def parse_request_data_and_get_user(request):
     try:
         db_user = get_user_by_username_or_email(user)
     except ObjectDoesNotExist:
-        raise ValueError(_('Student (username/email={user}) does not exist').format(user=user))
+        raise ValueError(_("We can't find the user you've entered. Make sure the username or email address is correct,"
```

Comment:
Is the user information now unavailable for use in the error message?  Unless the requirements were to make the message more generic, I think it would be useful to restate the username/email to provide more context for the problem.